contrib/mesos/pkg/minion/server.go

```diff
@@ -333,6 +336,10 @@ func (ms *MinionServer) AddExecutorFlags(fs *pflag.FlagSet) {
 
 	// hack to forward log verbosity flag to the executor
 	fs.Int32Var(&ms.logVerbosity, "v", ms.logVerbosity, "log level for V logs")
+	
+	// hacks to forward selected cadvisor flags to the executor
+	fs.StringVar(&ms.housekeepingInterval, "housekeeping_interval", ms.housekeepingInterval, "Interval between container housekeepings")
```

Comment:
@jdef I'm having trouble to run test-smoke.sh from my Mac.  I'll try attempt to run it from a Linux box.  Is this a known issue?

Another q, I got this error from the pre-commit hook:
Are you certain this flag should not have been declared with an - instead?
global_housekeeping_interval
housekeeping_interval

Should i change them to use global-housekeeping-interval and housekeeping-interval instead?  or shall i just update the excluded-flags.txt?  Thank you!